core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>We Need JavaScript in Your Browser <img src="/images/general/sad_face.png" /> </h2>
```

Comment:
With respect to the md-icons -- perhaps I was unclear, sorry. The material-design font is already included in the Oppia codebase. You just need to include the snippet of HTML I provided (the one starting and ending with 'i' tags) -- we already do similar things elsewhere, e.g. in core/templates/dev/head/exploration_editor/exploration_editor.html. No need to add a new folder etc.

With regard to the text, I agree that it's good not to blame the user. A possible alternative may be "Please Enable JavaScript :(", since that clearly and concisely tells the reader what to do. I'm not too bothered either way, though, so if you prefer it as-is, that's fine too!